Userland/cut.cpp

```diff
@@ -49,8 +60,20 @@ static void print_usage_and_exit(int ret)
 
 static void add_if_not_exists(Vector<Index>& indexes, Index data)
 {
-    auto find = [data](auto& other) { return other.m_from == data.m_from && other.m_to == data.m_to; };
-    if (indexes.find(find) == indexes.end()) {
+    auto min = [](ssize_t a, ssize_t b) { return a < b ? a : b; };
+    auto max = [](ssize_t a, ssize_t b) { return a > b ? a : b; };
```

Comment:
We already have `min()` and `max()` in `<AK/StdLibExtras.h>`, please use them instead of defining custom ones :)

Reply:
The AK library is so dense that is getting harder and harder to see what exists and what not. Maybe a small doxygen like documentation would help :P